Calculator/Calculator/Controller/CalculatorViewController.swift

```diff
@@ -14,149 +14,175 @@ class CalculatorViewController: UIViewController {
         super.viewDidLoad()
         
-        initializeUI()
+        resetUI()
     }
-    
-    private func initializeUI() {
-        operandLabel.text = CalculatorValue.zero
-        operatorLabel.text = CalculatorValue.emptyArray
-        scrollStackView.arrangedSubviews.forEach { $0.removeFromSuperview() }
+  
+    private func resetUI() {
+        if currentOperand.isEmpty {
+            operandLabel.text = CalculatorValue.zero
+        } else {
+            operandLabel.text = currentOperand
+        }
+        
+        operatorLabel.text = currentOperator
     }
     
     @IBAction private func numberButtonDidTap(_ sender: UIButton) {
         guard let number = sender.title(for: .normal) else { return }
         
-        if operandNumber.count < 20 {
-            operandNumber += number
-            operandLabel.text = numberFormatter(operand: operandNumber)
-        } else if operandNumber.contains(Character(CalculatorValue.decimalPoint)) && operandNumber.count < 20 {
-            operandNumber += number
-            operandLabel.text = operandNumber
+        if currentOperand.count < 20 {
+            currentOperand += number
+        
+            operandLabel.text = currentOperand.numberFormatter()
         }
     }
     
     @IBAction private func dotButtonDidTap(_ sender: UIButton) {
-        if operandNumber.isEmpty {
-            operandNumber = CalculatorValue.zeroDecimal
-            operandLabel.text = numberFormatter(operand: operandNumber)
-        } else if operandNumber.last == Character(CalculatorValue.decimalPoint) || operandNumber.count == 20 || operandNumber.contains(Character(CalculatorValue.decimalPoint)) {
-            operandLabel.text = numberFormatter(operand: operandNumber)
+        guard currentOperand.count < 20 &&
+                !currentOperand.contains(Character(CalculatorValue.decimalPoint)) else { return }
+   
+        if currentOperand.isEmpty || currentOperand == CalculatorValue.minus {
+            currentOperand += CalculatorValue.zeroDecimal
         } else {
-            operandNumber += CalculatorValue.decimalPoint
-            operandLabel.text = operandNumber
+            currentOperand += CalculatorValue.decimalPoint
         }
+        
+        operandLabel.text = currentOperand
     }
     
-    @IBAction private func operatorButtonDidTap(_ sender: UIButton) {
-        guard let operatorSymbol = sender.title(for: .normal) else { return }
+    private func addListStackView() -> UIStackView {
+        let operatorLabel = UILabel()
+        let operandLabel = UILabel()
+        let stackView = UIStackView()
         
-        if operandLabel.text == CalculatorValue.zero {
-            operatorLabel.text = operatorSymbol
+        stackView.spacing = 8
+        operandLabel.textColor = .white
+        operatorLabel.textColor = .white
+        operandLabel.font = UIFont.preferredFont(forTextStyle: .title3)
+        operatorLabel.font = UIFont.preferredFont(forTextStyle: .title3)
```

Comment:
가독성을 높여주기 위해 관련된 코드끼리 묶어주면 좋겠습니다!
```suggestion
        let stackView = UIStackView()
        stackView.spacing = 8
        
        let operandLabel = UILabel()
        operandLabel.textColor = .white
        operandLabel.font = UIFont.preferredFont(forTextStyle: .title3)
        
        let operatorLabel = UILabel()
        operatorLabel.textColor = .white
        operatorLabel.font = UIFont.preferredFont(forTextStyle: .title3)
```